runt.toml

```diff
@@ -30,6 +30,15 @@ cmd = """
 fud e -s cocotb.data {}.data --to cocotb-out {} -s calyx.flags ' -d canonicalize' -s filament.flags ' --show-models' -q
 """
 
+# [[tests]]
+# name = "ir-compile"
+# paths = [
+#   "tests/compile/*.fil"
+# ]
+# cmd = """
+# fud e -s cocotb.data {}.data --to cocotb-out {} -s calyx.flags ' -d canonicalize' -s filament.flags ' --show-models --ir' -q
+# """
```

Comment:
Why is this commented out?

Reply:
Most tests do not pass due to state.fil or parameters